plugins/woocommerce/src/Admin/Features/ProductBlockEditor/ProductTemplates/SimpleProductTemplate.php

```diff
@@ -0,0 +1,773 @@
+					'description' => __( 'This info will be displayed on the product page, category pages, social media, and search results.', 'woocommerce' ),
+				],
+			]
+		);
+		$basic_details->add_block(
+			[
+				'id'         => 'product-name',
+				'blockName'  => 'woocommerce/product-name-field',
+				'attributes' => [
+					'name'      => 'Product name',
+					'autoFocus' => true,
+				],
+			]
+		);
+		$basic_details->add_block(
+			[
+				'id'        => 'product-summary',
+				'blockName' => 'woocommerce/product-summary-field',
+			]
+		);
+		$pricing_columns  = $basic_details->add_block(
+			[
+				'id'        => 'product-pricing-columns',
+				'blockName' => 'core/columns',
+			]
+		);
+		$pricing_column_1 = $pricing_columns->add_block(
+			[
+				'id'         => 'product-pricing-column-1',
+				'blockName'  => 'core/column',
+				'attributes' => [
+					'templateLock' => 'all',
+				],
+			]
+		);
+		$pricing_column_1->add_block(
+			[
+				'id'         => 'product-regular-price',
+				'blockName'  => 'woocommerce/product-regular-price-field',
+				'attributes' => [
+					'name'  => 'regular_price',
+					'label' => __( 'List price', 'woocommerce' ),
+					'help'  => __( 'Manage more settings in <PricingTab>Pricing.</PricingTab>', 'woocommerce' ),
+				],
+			]
+		);
+		$pricing_column_2 = $pricing_columns->add_block(
+			[
+				'id'         => 'product-pricing-column-2',
+				'blockName'  => 'core/column',
+				'attributes' => [
+					'templateLock' => 'all',
+				],
+			]
+		);
+		$pricing_column_2->add_block(
+			[
+				'id'         => 'product-sale-price',
+				'blockName'  => 'woocommerce/product-sale-price-field',
+				'attributes' => [
+					'label' => __( 'Sale price', 'woocommerce' ),
+				],
+			]
+		);
+
+		// Description section.
+		$description_section = $general_group->add_section(
+			[
+				'id'         => 'product-description-section',
+				'attributes' => [
+					'title'       => __( 'Description', 'woocommerce' ),
+					'description' => __( 'What makes this product unique? What are its most important features? Enrich the product page by adding rich content using blocks.', 'woocommerce' ),
+				],
+			]
+		);
+		$description_section->add_block(
+			[
+				'id'        => 'product-description',
+				'blockName' => 'woocommerce/product-description-field',
+			]
+		);
+		// Images section.
+		$images_section = $general_group->add_section(
+			[
+				'id'         => 'product-images-section',
+				'attributes' => [
+					'title'       => __( 'Images', 'woocommerce' ),
+					'description' => sprintf(
+					/* translators: %1$s: Images guide link opening tag. %2$s: Images guide link closing tag. */
+						__( 'Drag images, upload new ones or select files from your library. For best results, use JPEG files that are 1000 by 1000 pixels or larger. %1$sHow to prepare images?%2$s', 'woocommerce' ),
+						'<a href="http://woocommerce.com/#" target="_blank" rel="noreferrer">',
+						'</a>'
+					),
```

Comment:
Any reason why `sprintf` was used instead of `createInterpolateElement`?

Reply:
I am not sure, I think this would be a good follow up as it would also require changes in the `Tab` block. Although I have mixed feelings on this myself, as it seems more appropriate to handle this here, instead of having to be aware of what components are available through `createInterpolateElement` in the specific block.